packages/wonder-blocks-button/src/components/button-core.js

```diff
@@ -2,7 +2,7 @@
 import * as React from "react";
 import {StyleSheet} from "aphrodite";
 import {Link} from "react-router-dom";
-import * as PropTypes from "prop-types";
```

Comment:
oh I totally missed that! Kinda related question: why do we have `prop-types` as a peer dep in some of the packages?

Reply:
It's no longer needed.  Thanks for flagging this.  I've removed it as a peerDep now.